test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+import boto3
+import pytest
+
+from test.test_utils import get_repository_and_tag_from_image_uri, LOGGER
+
+
+def test_dlc_major_version_label(image, region):
```

Comment:
nit: add docstring

also you can add a marker for the integration `@pytest.mark.integration('dlc_major_version_label')` or something

Reply:
We didn't have markers on other sanity tests, so didn't add one here. Can add.